src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ThemeServicePermissionTests.cs

```diff
@@ -42,7 +42,7 @@ await using (var context = DbUtils.InMemoryApplicationDbContext(contextId))
                 await context.SaveChangesAsync();
             }
 
-            PolicyCheckBuilder()
+            await PolicyCheckBuilder()
```

Comment:
Nice one - these ones creep in from time to time - hopefully if we can eliminate all warnings from the codebase then we'll just be left with legitimate ones so we can spot these appearing.